docs/user/rollout-policies.md

```diff
@@ -0,0 +1,107 @@
+# Rollout Device Selection and Rollout Disruption Budget
+
+## Overview
+
+When performing a rollout using `flightctl`, it is crucial to manage which devices participate in the rollout and how much disruption is acceptable. This document outlines the device selection process and the concept of the rollout disruption budget to ensure controlled and predictable rollouts.
+
+---
+
+## Rollout Device Selection
+
+### **Device Targeting**
+
+A rollout applies only to devices that belong to a fleet. Each device can belong to only a single fleet. Since rollout definitions are done at the fleet level, the selection process determines which devices within a fleet will participate in the rollout based on label criteria.
```

Comment:
"which devices within a fleet will participate in the rollout" - same as above - all devices in a fleet will participate.